templates/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Registration Form</title>
+    <!-- Вставьте любые стили или скрипты, которые вам нужны -->
+</head>
+<body>
+
+<h1>Registration Form</h1>
+
+<form method="POST" action="{{ url_for('register') }}">
+    {{ form.hidden_tag() }}
+    
+    <label for="name">Name:</label>
+    {{ form.name() }}
+    <br>
+
+    <label for="surname">Surname:</label>
+    {{ form.surname() }}
+    <br>
+
+    <label for="email">Email:</label>
+    {{ form.email() }}
+    <br>
+
+    <label for="password">Password:</label>
+    {{ form.password() }}
+    <br>
+
+    <button type="submit">Register</button>

```

Comment:
А тут много повторений. Нужно привести код в соответствии с принципом DRY
```suggestion
{{ form.csrf_token }}
<p>
    {% for field in form if field.name != 'csrf_token' %}
    <p>
        {{ field.label }}<br>
        {{ field }}
    </p>
    <p>
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
    </p>
    {% endfor %}
</p>
```
Причем строки
```html
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
```
будут повторять flash сообщение об ошибке. Повторюсь, что вы можете решить, что лучше: либо выводить сообщение тут, либо выше.